lib/Notification/Notifier.php

```diff
@@ -385,7 +385,13 @@ protected function parseCall(INotification $notification, Room $room, IL10N $l)
 			} else {
 				$notification
 					->setParsedSubject($l->t('A group call has started'))
-					->setRichSubject($l->t('A group call has started'));
+					->setRichSubject($l->t('A group call has started'), [
+						'call' => [
```

Comment:
this is "invalid", because the `{call}` placeholder is not used in the message.

I guess you do this, so you have the room details available in the mobile apps?

Reply:
Yes, indeed.